files/qalter.pl

```diff
@@ -104,6 +107,14 @@ sub qalter_main
         qalter_update(\%update, 'requeue')
     }
 
+    # Use Slurm's Perl API to change the reservation
+    if ($additional_attributes) {
+        if ($additional_attributes =~ m/x(?ii)="?ADVRES:([^\r\n\t\f\v "]+)/) {
```

Comment:
what's with the regexp? `[\w-.]`? not sure what is allowed though

Reply:
the problematic part that space and even `"` can be in the reservation name, but torque accepts `x="advres:reservationname"` or `x=advres:reservationname` formats. Otherwise it seems that anything goes, but programming wise that could be problematic, so this is why excluded only whithespaces and `"`
Just tried, slurm accepted to make a `ReservationName=test test"@#$%^&*!`

Reply:
ok, thx for the info